src/lib/common/statistics.cpp

```diff
@@ -211,5 +214,8 @@ void statisticsUpdate(RequestType request, Format inFormat)
   case EntitiesResponse:                                 ++noOfEntitiesResponses; break;
   case EntryPointsRequest:                               ++noOfEntryPointsRequests; break;
   case EntryPointsResponse:                              ++noOfEntryPointsResponses; break;
+  case EntityRequest:                                    ++noOfEntityRequests; break;
+  case EntityResponse:                                   ++noOfEntityResponses; break;
+
```

Comment:
I think that was meant for a further issue improving functionality.  AFAICS,  /v2/entities does not implement it yet, likely for the same reason ... To be confirmed by @fgalan